### ISSUE TITLE
hyper-parameters

### ISSUE THREAD

**Issue by chenxiao18481132** [mperezcarrasco/Pytorch-VaDE#2]
I run your code in my computer, but I didn't get the good results as your figure (acc about 80%). Do I need to reset the hyper-parameters?

**Comment by mperezcarrasco** [comment]
In my experience running this code (and even the original code released by the authors), the results are highly related to the initialization of the VaDE (i.e the pretrained autoencoder). Try re-running the code. Sometimes it will give you  ~90% and sometimes ~80%.
I did not perform a complete analysis, but I guess that if you train the autoencoder using a validation set and you save the best model within several epochs, you will reach better results.
